### PR TITLE
RSM project directory underlying bRSM paper

### PR DESCRIPTION
**Pull request by onejgordon**

### CODE REVIEW

**Comment by subutai** [comment]
@onejgordon We're looking into the failure - it's a nupic.torch versioning issue.

**Comment by lscheinkman** [comment]
@onejgordon the pytorch dependency failure was fixed. You should run `flake8`, `isort` or `black` to fix the style issues

**Comment by onejgordon** [comment]
Thanks @lscheinkman @subutai. Style fixes in progress.

**Comment by onejgordon** [comment]
Ready for review/merge.

File: requirements.txt [hunk 1]
```diff
@@ -40,5 +40,4 @@ requests
 scikit-image
 seaborn>=0.9.0
 tabulate
-torchvision
-tqdm
```
Comment (lscheinkman) [inline requirements.txt]: Why did you remove `tqdm` dependency?
It is used by nupic.research frameworks and projects.

Comment (onejgordon) [reply requirements.txt]: Must have happened at some point in a merge conflict resolution. requirements.txt shouldn't be touched in this PR. Updating.

Comment (lscheinkman) [reply requirements.txt]: 👍

**Comment by onejgordon** [comment]
Thanks. Conflict resolved.
